string[] plant_paths
geometry_msgs/Pose[] ghost_configs
---